cirq/circuits/circuit_test.py

```diff
@@ -27,6 +27,40 @@
 import cirq.google as cg
 
 
+def raise_value_error_if_wrong_type(obj, classinfo):
```

Comment:
I think defining this function is needless generalization and actually makes the code less readable.